src/test/java/com/st/utopia/cancellationservice/service/TicketServiceTest.java

```diff
@@ -0,0 +1,40 @@
+package com.st.utopia.cancellationservice.service;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.junit4.SpringRunner;
+
+
+import com.st.utopia.cancellationservice.model.Ticket;
+
+
+@RunWith(SpringRunner.class)
+@SpringBootTest
+public class TicketServiceTest {

```

Comment:
Is your test coverage up to the level we are required to meet?

Reply:
yes the test coverage is up to the level we are required to meet, 70%.
I didn't want test all the DAO methodes because since the basic ones, runs perfectly, the add and find and update so for sure the other shall work fine

Reply:
Right; my question was about the project as a whole, and I expected that the test coverage would be above the minimum, but I wanted to make sure.